test/unit/math/prim/mat/fun/chol2inv_test.cpp

```diff
@@ -0,0 +1,54 @@
+#include <stan/math/prim/mat.hpp>
```

Comment:
We want derivative tests for rev, fwd, and mix modes.  Here you have a second way to evaluate the function so you can just test that the derivatives are the same for a couple simple cases.